chain-core/src/init/address.rs

```diff
@@ -33,14 +33,16 @@ use crate::common::{H256, HASH_SIZE_256};
 pub enum CroAddressError {
     // TODO: use directly bech32::Error or wrap it
     Bech32Error(String),
+    InvalidNetwork,
     ConvertError,
 }
 
 #[cfg(feature = "bech32")]
 impl ::std::error::Error for CroAddressError {}
 
-// CRMT: mainnet transfer
-// CRTT: testnet transfer
+// CRO: mainnet transfer
+// TCRO: testnet transfer
+// DCRO: testnet transfer
```

Comment:
DCRO = devnet/regnet (for unit tests or local development testing)
TCRO = future deployed public Testnet